resources/language/resource.language.en_gb/strings.po

```diff
@@ -811,3 +811,35 @@ msgstr ""
 msgctxt "#30198"
 msgid "Exporting new episodes"
 msgstr ""
+
+msgctxt "#30199"
+msgid "Shared library (Need Kodi MySql server database)"
+msgstr ""
+
+msgctxt "#30200"
+msgid "Use MySQL shared library database"
+msgstr ""
+
+msgctxt "#30201"
+msgid "Username"
+msgstr ""
+
+msgctxt "#30202"
+msgid "Password"
+msgstr ""
+
+msgctxt "#30203"
+msgid "Host IP"
+msgstr ""
+
+msgctxt "#30204"
+msgid "Port"
+msgstr ""
+
+msgctxt "#30205"
+msgid "Test database connection"
+msgstr ""
+
+msgctxt "#30206"
+msgid "ERROR: The MySQL database is not reachable - check the connection - fallback to local database"
```

Comment:
Are we really falling back?

Reply:
yes if mysql not work instantiates local mysql sqlite database,
when the connection is corrected, resume with mysql
if you prefer to improve the error phrase, propose one

Reply:
now that I think better, there is a problem
the addon restarts when an action occurs so the connection when working is automatically restored, but in the service remains the old one once initializated ....mmmh....

Reply:
I think the easiest way is that in case of connection problems we disable via code the setting
"use_mysql" to False and so fallback to sqlite.
this way when the user tries to reactivate the mysql connection via settings
it is also updated in the service (after the change to be made)

Reply:
I try to do something then when you have time you let me know if it could be fine

Reply:
I forgot global is already reinitialized when settings change so g.SHARED_DB shoud be yet changed... now I am confused i wait your rpi log